test/test_device.py

```diff
@@ -662,6 +662,14 @@ def test_display_address_multisig_descriptor(self):
         sh_wsh_multi_desc = sh_wsh_multi_desc.replace("'", "h").split('#')[0]
         wsh_multi_desc = wsh_multi_desc.replace("'", "h").split('#')[0]
 
+        # descriptor with xpubs
+        if self.full_type == 'trezor_t':
+            account_xpub = self.do_command(self.dev_args + ['getxpub', 'm/48h/0h/0h/2h'])['xpub']
+            result = self.do_command(self.dev_args + ['displayaddress', '--desc', 'wsh(multi(2, [' + self.fingerprint + '/48h/0h/0h/2h]' + account_xpub + '/0/0,[' + self.fingerprint + '/48h/0h/0h/2h]' + account_xpub + '/1/0))'])
+            self.assertNotIn('error', result)
+            self.assertNotIn('code', result)
+            self.assertIn('address', result)
```

Comment:
For a sanity check, can you put the descriptor through the `deriveaddresses` RPC and check that bitcoind returns the same address as the device?

Reply:
Sure, added. Let me know if that works for you.